README.md

```diff
@@ -39,22 +39,29 @@ recommendations are documented at [voxpupuli.org](https://voxpupuli.org/docs/rev
 ```puppet
 class foo (
   String $bar = '',
+  String[0] $foo = '',
+  Variant $baz = '',
 ) {
   # logic
 }
 ```
 
+All four of the above variantions aren't recommended.
```

Comment:
It made me think of https://www.youtube.com/watch?v=jk3EsXgXcyQ, but in this case there are 3 ;)

Reply:
fixed it